src/siwe.rs

```diff
@@ -289,19 +302,19 @@ impl AuthorizationPolicy<SIWETokens> for OrbitMetadata {
         };
 
         let invoker = format!(
-            "did:pkh:eip155:{}:0x{}#blockchainAccountId",
+            "did:pkh:eip155:{}:{}#blockchainAccountId",
             &t.invocation.chain_id,
-            &hex::encode(&t.invocation.address)
+            &to_checksum(&H160(t.invocation.address), None)
```

Comment:
the did:pkh spec doesnt specify EIP55 account encoding (the `eip155` example contains lowercase hex) however for now we can probably use this as the standard `address` value given by wallets seems to always be EIP55 encoded.

Reply:
this is in the process of being addressed in the SSI CAIP-10 implementation